flexmeasures/data/services/resources.py

```diff
@@ -53,19 +54,42 @@ def get_assets(
     return _build_asset_query(owner_id, order_by_asset_attribute, order_direction).all()
 
 
+def get_sensors(
+    owner_id: Optional[int] = None,
+    order_by_asset_attribute: str = "id",
+    order_direction: str = "desc",
+) -> List[Asset]:
+    """Return a list of all Sensor objects owned by current_user's organisation account
+    (or all users or a specific user - for this, admins can set an owner_id).
+    """
+    # todo: switch to using authz from https://github.com/SeitaBV/flexmeasures/pull/234
+    return [
+        asset.corresponding_sensor
+        for asset in get_assets(owner_id, order_by_asset_attribute, order_direction)
+    ]
+
+
 def has_assets(owner_id: Optional[int] = None) -> bool:
     """Return True if the current user owns any assets.
     (or all users or a specific user - for this, admins can set an owner_id).
     """
     return _build_asset_query(owner_id).count() > 0
 
 
-def can_access_asset(asset: Asset) -> bool:
-    """Return True if the current user is an admin or the owner of the asset:"""
+def can_access_asset(asset_or_sensor: Union[Asset, Sensor]) -> bool:
```

Comment:
As I see it this is only used in API endpoints. I'd like a note that we'll deprecate this way of checking soon.

First, we'll change this method so it uses the new auth model  in the case of `asset_or_sensor` being a `Sensor`.

Later, when we have faded out the old classes, the USEF-style API endpoints can move over to using the new auth decorators. Does this require its own card it [Project 11](https://github.com/SeitaBV/flexmeasures/projects/11) or is this covered by moving over to API v3 completely?

In any case, then this can go.

Reply:
I added 3 todos in the code, and made a new card for project 11.